main.go

```diff
@@ -824,6 +831,10 @@ func main() {
              </body>
              </html>`))
 	})
-	log.Infoln("Starting HTTP server on", *listenAddress)
-	log.Fatal(http.ListenAndServe(*listenAddress, nil))
+
+	level.Info(logger).Log("msg", "Listening on address", "address", *listenAddress)
```

Comment:
What's the motivation to remove the information that the HTTP server is being started? Minor thing, but `msg="Listening on address", address="localhost:80"` stutters.

Reply:
To just make it uniform across all the projects